packages/custom-elements/externs/custom-elements.d.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @externs
```

Comment:
here and elsewhere: I don't think this file should be marked as `@externs`. In the internal bazel rules, a ts_declaration() automatically produces externs

Reply:
Removed.